<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>view_Categories</class>
 <widget class="QMainWindow" name="view_Categories">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mes Catégories</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>471</width>
      <height>331</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="200">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="200">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>240</red>
          <green>240</green>
          <blue>240</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="rowCount">
     <number>0</number>
    </property>
    <property name="columnCount">
     <number>3</number>
    </property>
    <column/>
    <column/>
    <column/>
   </widget>
   <widget class="QLineEdit" name="edit_nom_cat">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>70</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_add_cat">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>110</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Ajouter</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-10</y>
      <width>891</width>
      <height>481</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>0</y>
      <width>401</width>
      <height>61</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>120</red>
          <green>120</green>
          <blue>120</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>30</pointsize>
      <weight>50</weight>
      <bold>false</bold>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string>Mes Catégories</string>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>tableWidget</zorder>
   <zorder>edit_nom_cat</zorder>
   <zorder>btn_add_cat</zorder>
   <zorder>label_3</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionQuitter"/>
   </widget>
   <widget class="QMenu" name="menuNavigation">
    <property name="title">
     <string>Navigation</string>
    </property>
    <addaction name="actionAccueil"/>
    <addaction name="separator"/>
    <addaction name="actionComptes"/>
    <addaction name="actionMagasins"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuNavigation"/>
  </widget>
  <action name="actionAccueil">
   <property name="text">
    <string>Accueil</string>
   </property>
  </action>
  <action name="actionMagasins">
   <property name="text">
    <string>Magasins</string>
   </property>
  </action>
  <action name="actionComptes">
   <property name="text">
    <string>Comptes</string>
   </property>
  </action>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
  <action name="actionRecherche">
   <property name="text">
    <string>Recherche</string>
   </property>
  </action>
  <action name="actionCat_gories">
   <property name="text">
    <string>Catégories</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
